api/web3server.go

```diff
@@ -344,6 +343,10 @@ func (svr *Web3Server) getBlockByNumber(in interface{}) (interface{}, error) {
 
 	blkMetas, err := svr.coreService.BlockMetas(num, 1)
 	if err != nil {
+		st, ok := status.FromError(err)
+		if ok && st.Code() == codes.NotFound {
```

Comment:
`codes.InvalidArgument`

Reply:
there are also 2 other places calling `coreService.BlockMetas` in this file, do those 2 places need this fix?

Reply:
none